lms/static/sass/discussion/_discussion.scss

```diff
@@ -423,36 +261,6 @@ body.discussion {
       word-wrap: break-word;
     }
 
-    h1 {
```

Comment:
Context: These styles (426-455) were also removed from this deprecated partial and into _thread.scss under the "thread - elements - shared styles" section. Instead of using the h1 element it now uses the appropriate (new) classes (.discussion-post, .discussion-response, .discussion-comment)